app/konnect/gateway-manager/data-plane-nodes/azure-peering.md

```diff
@@ -0,0 +1,88 @@
+---
+title: How to configure Azure VNET Peering
+---
+
+
+
+## How does VNET Peering Work
```

Comment:
I think this is missing the what and why. I don't know what it is or why I would use it.